src/Database/QueryCompiler.php

```diff
@@ -97,6 +97,17 @@ public function compile(Query $query, ValueBinder $generator)
             $this->_sqlCompiler($sql, $query, $generator),
             $this->{'_' . $type . 'Parts'}
         );
+
+        // Propagate bound parameters from sub-queries if the
+        // placeholders can be found in the SQL statement.
+        if ($query->valueBinder() !== $generator) {
+            foreach ($query->valueBinder()->bindings() as $binding) {
+                $placeholder = ':' . $binding['placeholder'];
```

Comment:
The chances for finding this string in a query a very high. For example finding `:c1` in `select ... :c14` will return true.

I wonder if there is a better way to do this, do we actually need to search for the placeholder or can we just trust it is there since it was bound already?